### PR TITLE
Migrate global migration to Vanilla v1 in /about section

### LINKED ISSUE #1278: Replicate existing navigation pattern with v1

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Migrate global migration to Vanilla v1 in /about section

## QA

- Pull code
- Run `./run serve`
- Go to file `templates/about/index.html`
- Change line 3 to `{% extends "about/base_about-v1.html" %}`
- Change line 8 to `{% include "templates/_nav_breadcrumb-v1.html" with section_title="About" page_title="The Ubuntu Project" %}`
- View [http://0.0.0.0:8001/about](http://0.0.0.0:8001/about) across breakpoints
- Check code

**Important notes:**

- This is a first pass, `styles-v1.scss` should be viewed as a WIP throughout Vanilla v1 migration
- This is a blocking PR so due to expediency, I have not yet split the different code overrides into separate partials. These are commented as @TODOs and will be moved out in due course.
- Site search in nav is currently hidden across all breakpoints, this will be completed as a separate issue

## Issue / Card

Fixes #1278

### CODE REVIEW

**Comment by deadlight** [comment]
I'll note some differences in the look, it'll be up to design to make a call on them:

 - The font size used in the nav is larger - and the nav bar is taller to accommodate this.
 - There is no dropshadow under the main nav
 - There is no border under the secondary nav bar
 - Secondary nav hover state is underlined whereas the existing style is a colour change on hover with no underline.

Can you also pull in the global nav styling. This will be unchanged as it's used as-is across all sites.

File: static/sass/styles-v1.scss [hunk 1]
```diff
@@ -1,5 +1,242 @@
 @charset "UTF-8";
 
+$color-brand: #e95420;
```
Comment (deadlight) [inline static/sass/styles-v1.scss]: Could we move these to a duplicate global settings file?

File: templates/download/_nav_tertiary-v1.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
+    <ul class="third-level-nav">
+{% if level_2 == 'server' %}
+        <li><a {% if level_3 == 'arm' %} class="active"{% endif %}href="/download/server/arm">ARM</a></li>
```
Comment (deadlight) [inline templates/download/_nav_tertiary-v1.html]: Fix indentation in this file, please.

File: templates/internet-of-things/_nav_secondary-v1.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+  <ul{% if list_class %} class="{{ list_class }}"{% endif %} class="hover-menu">
+    <li><a{% if level_1 == 'internet-of-things' and not level_2 %} class="active"{% endif %} href="/internet-of-things">Overview</a></li>
+    <li><a{% if level_2 == 'digital-signage' %} class="active"{% endif %} href="/internet-of-things/digital-signage">Digital signage</a></li>
+    <li><a{% if level_2 == 'robotics' %} class="active"{% endif %} href="/internet-of-things/robotics">Robotics</a></li>
+        <li><a{% if level_2 == 'gateways' %} class="active"{% endif %} href="/internet-of-things/gateways">Gateways</a></li>
```
Comment (deadlight) [inline templates/internet-of-things/_nav_secondary-v1.html]: Fix line indent

File: templates/templates/base-v1.html [hunk 4]
```diff
@@ -52,8 +52,7 @@
     <meta name="google-site-verification" content="ddh2iq7ZuKf1LpkL_gtM_T7DkKDVD7ibq6Ceue4a_3M" />
   </head>
 
-  <body class="{% if level_1 != "" %}{% if level_1 == '2012' %}body-{% endif %}{{ level_1 }}{% endif %}{% if level_2 %} {% if level_1 == '2012' %}body-{% endif %}{{ level_1 }}-{{ level_2 }}{% endif %}{% if level_3 %} {% if level_1 == '2012' %}body-{% endif %}{{ level_1 }}-{{ level_2 }}-{{ level_3 }} {% endif %} {% include "templates/body-class.html" %}{% if extra_body_class != "" %} {% block extra_body_class %}{% endblock %}{% endif %}">
-
+  <body>
```
Comment (deadlight) [inline templates/templates/base-v1.html]: Is the class .body-[level 1] not targetted anywhere anymore?

Comment (barrymcgee) [inline templates/templates/base-v1.html]: Not within our new v1 styles.

**Pull request merged**